src/test/java/SignUp.java

```diff
@@ -1,80 +1,107 @@
 import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
-import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.support.ui.ExpectedConditions;
 import org.testng.Assert;
-import org.testng.annotations.AfterMethod;
-import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
+
+import java.sql.Timestamp;
 import java.util.Random;
-import java.util.concurrent.TimeUnit;
 
 public class SignUp extends BaseTest{
 
 
     @Test
     public static void signUpSuccess() {
-        driver.findElement(By.xpath("//div[@class='styled__Wrapper-sc-5elvgs-0 gTjIWz']/div/div[3]")).click();
+        getPage();
+        setEmailField(getRandomEmail());
+        setPassword("incorrectPassword");
+        setConfirmPassword("incorrectPassword");
+        setZipCode("90020");
+        clickCheckboxTermsAndConditions();
+        clickCheckboxHIPAA();
+        clickToSignUp();
 
-        Random randomGenerator = new Random();
-        int randomInt = randomGenerator.nextInt(1000);
+        WebElement phoneNumber = driver.findElement(By.cssSelector("[content='Confirm Phone Number']"));
+        Assert.assertTrue(phoneNumber.isDisplayed());
+    }
 
-        driver.findElement(By.cssSelector("[placeholder='Email']")).sendKeys("almazing" + randomInt + "@gmail.com");
-        driver.findElement(By.cssSelector("[placeholder='Password']")).sendKeys("testingPassword");
-        driver.findElement(By.cssSelector("[placeholder='Confirm Password']")).sendKeys("testingPassword");
-        driver.findElement(By.cssSelector("[placeholder='Zipcode']")).sendKeys("testing");
+    @Test
+    public static void signUpWrongEmail() {
+        getPage();
+        setEmailField("gmail.com");
+        setPassword("incorrectPassword");
+        setConfirmPassword("setConfirmPassword");
+        setZipCode("90020");
+        clickCheckboxTermsAndConditions();
+        clickCheckboxHIPAA();
+        clickToSignUp();
 
-        WebElement checkBoxOne = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("[class='styled__CheckboxWrapper-sc-1x07d2p-2 cjrWjQ'] [class*='ws-checkbox']")));
-        checkBoxOne.click();
+        WebElement errorAlert = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("[content='Oops!']")));
```

Comment:
It would be better move this wait into the method with name "waitElementVisibility"and place it in "Base" class and then you can use it in other classes without duplicating wait.until for each element.